locksmith/migrations/20190318152733-create-user.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Users', {
+      publicKey: {
+        type: Sequelize.STRING,
+        primaryKey: true,
+        allowNull: false,
+        unqiue: true
+      },
+      recoverySeed: {
```

Comment:
I am not sure `seed` is the right word here. `recoveryPhrase` is probably more accurate? Arguably, a seed is something which "generates" something else. This is used to 'encrypt' the private key.